tests/phpunit/elementor/modules/atomic-widgets/styles/test-styles-renderer.php

```diff
@@ -620,6 +621,7 @@ public function test_render__style_with_background_image_transformers_without_im
                         ],
 
                         'meta' => [],
+
```

Comment:
I would try to avoid such unnecessary changes.

It is a good practice to review your own PRs and clean up all these small kind of things. I have learned a lot from doing that.